test/cli/rbi-gen-single/rbi-gen-single.out

```diff
@@ -6,8 +6,11 @@
 class Family::Simpsons < Object
   sig {returns(Bart::Character)}
   def bart; end
+  sig {returns(T.nilable(Bart::Character))}
+  def no_bart; end
```

Comment:
I don't think it's a deal-breaker that the alias didn't stick around, but I'll at least figure out why it's happening.